test/test-infra/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Dockerfile used by out prow jobs.
+# # The sole purpose of this image is to customize the command run.
+FROM gcr.io/mlkube-testing/test-worker:latest
```

Comment:
I don't think you need a Dockerfile. There's no reason we should be building our own imag.e